src/Microsoft.AspNetCore.SignalR/HubContext.cs

```diff
@@ -20,6 +22,11 @@ public HubContext(HubLifetimeManager<THub> lifetimeManager)
 
         public virtual IGroupManager Groups { get; }
 
+        public IClientProxy AllExcept(List<string> excludedIds)
```

Comment:
Should use `IReadOnlyList<string>` or `IEnumerable<string>` here. We never plan to write to this list, and we don't care exactly how it's implemented. In fact, if our implementations are going to use `HashSet<string>`, `IEnumerable` would be better since we're going to need to convert it to a different type of collection anyway.